pytorch_lightning/loggers/base.py

```diff
@@ -60,405 +33,9 @@ class LightningLoggerBase(ABC):
-    """
-
-    def __init__(self, logger_iterable: Iterable[LightningLoggerBase]):
-        super().__init__()
-        self._logger_iterable = logger_iterable
-
-    def __getitem__(self, index: int) -> LightningLoggerBase:
-        return list(self._logger_iterable)[index]
-
-    def after_save_checkpoint(self, checkpoint_callback: "ReferenceType[ModelCheckpoint]") -> None:
-        for logger in self._logger_iterable:
-            logger.after_save_checkpoint(checkpoint_callback)
-
-    def update_agg_funcs(
-        self,
-        agg_key_funcs: Optional[Mapping[str, Callable[[Sequence[float]], float]]] = None,
-        agg_default_func: Callable[[Sequence[float]], float] = np.mean,
-    ):
-        for logger in self._logger_iterable:
-            logger.update_agg_funcs(agg_key_funcs, agg_default_func)
-
-    @property
-    def experiment(self) -> List[Any]:
-        """Returns a list of experiment objects for all the loggers in the logger collection."""
-        return [logger.experiment for logger in self._logger_iterable]
-
-    def agg_and_log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None):
-        for logger in self._logger_iterable:
-            logger.agg_and_log_metrics(metrics=metrics, step=step)
-
-    def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) -> None:
-        for logger in self._logger_iterable:
-            logger.log_metrics(metrics=metrics, step=step)
-
-    def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
-        for logger in self._logger_iterable:
-            logger.log_hyperparams(params)
-
-    def log_graph(self, model: "pl.LightningModule", input_array=None) -> None:
-        for logger in self._logger_iterable:
-            logger.log_graph(model, input_array)
-
-    def log_text(self, *args, **kwargs) -> None:
-        for logger in self._logger_iterable:
-            logger.log_text(*args, **kwargs)
-
-    def log_image(self, *args, **kwargs) -> None:
-        for logger in self._logger_iterable:
-            logger.log_image(*args, **kwargs)
-
-    def save(self) -> None:
-        for logger in self._logger_iterable:
-            logger.save()
-
-    def finalize(self, status: str) -> None:
-        for logger in self._logger_iterable:
-            logger.finalize(status)
-
-    def close(self) -> None:
-        """
-        .. deprecated:: v1.5
-            This method is deprecated in v1.5 and will be removed in v1.7.
-            Please use `LoggerCollection.finalize` instead.
-        """
+    def __init__(self, *args, **kwargs) -> None:  # type: ignore[no-untyped-def]
         rank_zero_deprecation(
-            "`LoggerCollection.close` method is deprecated in v1.5 and will be removed in v1.7."
-            " Please use `LoggerCollection.finalize` instead."
+            "The `pl.loggers.base.LightningLoggerBase` is deprecated. "
+            " Please use `pl.loggers.logger.Logger` instead."
```

Comment:
```suggestion
            "`pytorch_lightning.loggers.base.LightningLoggerBase` is deprecated. "
            " Please use `pytorch_lightning.loggers.logger.Logger` instead."
```

Reply:
In addition, should we note that this deprecated class will be removed in v1.8?